### PR TITLE
Lower ThriftClientManager log level

### PR DESCRIPTION
**Pull request by critical27**
The invalid channel log will print logs forever, especially for raft client. (chaos test will kill storage)

### CODE REVIEW

File: src/common/base/SlowOpTracker.h [hunk 1]
```diff
@@ -30,7 +30,7 @@ class SlowOpTracker {
     }
 
     void output(const std::string& prefix, const std::string& msg) {
-        LOG(INFO) << prefix << ", total time:" << dur_ << "ms, " << msg;
+        LOG(INFO) << prefix << "total time:" << dur_ << "ms, " << msg;
```
Comment (panda-sheep) [inline src/common/base/SlowOpTracker.h:33]: why change here？

**Comment by dutor** [comment]
What do you mean by _forever_?

File: src/common/base/SlowOpTracker.h (hunk 1 above)
Comment (critical27) [reply src/common/base/SlowOpTracker.h:33]: There are one extra comma here, `I0330 15:39:30.527143 16723 SlowOpTracker.h:33] [Port: 27001, Space: 33, Part: 95] , total time:53ms, Total send logs: 17`

**Comment by critical27** [comment]
> What do you mean by _forever_?

If kill one storage node in 3 replica, leader will try to append logs to follower, the log will become quite big, several gigabyte.

**Comment by dutor** [comment]
> > What do you mean by _forever_?
> 
> If kill one storage node in 3 replica, leader will try to append logs to follower, the log will become quite big, several gigabyte.

Roger that.

Meanwhile, consider the `LOG_EVERY_N` family.

File: src/common/base/SlowOpTracker.h (hunk 1 above)
Comment (panda-sheep) [reply src/common/base/SlowOpTracker.h:33]: Careful enough！

**Review by panda-sheep — APPROVED** [review]
well done！

**Review by bright-starry-sky — APPROVED** [review]

**Pull request merged**